### DISCUSSION TITLE
How do I --clean-after-build in Manifest mode

### DISCUSSION THREAD

**Discussion by diehard2** [microsoft/vcpkg#30865] [General]
I'm running out of room on my github runner. is there a way to --clean-after-build  in manifest mode?

**Comment by netcorefan1** [comment]
Not sure if this is a supported option, but did you tried to append `-cleanafterbuild` to `VCPKG_FEATURE_FLAGS` environment variable?

**Comment by dg0yt** [comment]
You can pass arguments to to vcpkg via the `VCPKG_INSTALL_OPTIONS` cmake variable.
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration#vcpkg_install_options

(But remember that "vcpkg-affecting variables must be defined before the first `project()` directive, such as via the command line or `set()` statements.")
